bigflow/bigquery/dataset_manager.py

```diff
@@ -40,11 +40,12 @@ def decorated(*args, **kwargs):
 
 def get_partition_from_run_datetime_or_none(run_datetime):
     """
-    :param run_datetime: string run datetime in format YYYY-MM-DD HH:mm:ss or YYY-MM-DD
-    :return: string partition in format YYYYMMDD
+    :param run_datetime: string run datetime in format YYYY-MM-DD HH:mm:ss or YYYY-MM-DD
+    :return: string partition in format YYYYMMDDHH for run_datetime in format: YYYY-MM-DD HH:mm:ss or
+     YYYYMMDD for run_datetime in format YYYY-MM-DD
     """
     if run_datetime is not None:
```

Comment:
nit: `if run_datetime is not None:` => `if run_datetime:`

Reply:
Done